bicycleparameters/cycle_app.py

```diff
@@ -128,8 +128,10 @@
                                                      },
                                                      style_data_conditional=[
                                                          {
-                                                             'if': {'column_editable': False},
-                                                             'cursor': 'not-allowed'
+                                                            "if": {"state": "selected"},              # 'active' | 'selected'
```

Comment:
Don't leave commented old code like this. It is in the git history, so you can always get it back.